### ISSUE TITLE
Paladin Censure Spell? Instant 5-stack DoT

### ISSUE THREAD

**Issue by majkefer** [gamefreedomgit/Maelstrom#3017]
**Description:**
When I skipped my Pala to level 80, I've noticed that there's a spell available that's not in the spellbook.
I have tried to bind it with a macro and realised that it's probably not supposed to be available like this.
The spell is called "Censure" which is a passive effect from Seal of Truth that does DoT.
The problem is that you can bind it and use it without any cooldown or mana cost and stack up Censure stacks instantly to 5.
Works even if you're not using Seal of Truth at the moment.

**How to reproduce:**
Basically you make a macro
#showtooltip
/cast Censure
Now you can spam this macro to insta 5-stack Censure DoT on a target.

Can't attach the video, file is too big. So here's a link:
https://www.youtube.com/watch?v=A6XfOqHwX6s&ab_channel=majkefer

**How it should work:**
Most likely shouldn't be available to the player as a spell, but rather only from SoT passive.

**Database links:**
-

**Comment by UltraNix** [comment]
Fixed.

**Comment by bonfax4** [comment]
@UltraNix This needs to be revisited. Saw a pala doing 50k dps single target dps out of using this bug with seal of command.
